src/utils.ts

```diff
@@ -478,6 +479,95 @@ export async function handleExternalProject(
   }
 }
 
+/**
+ * Config External CMake Project config file as an IoT Workbench Container
+ * Project. Throw cancel operation error if not CMake project. Update project
+ * host type and board id in IoT Workbench project file.
+ * @param scaffoldType
+ */
+export async function configExternalCMakeProjectToIoTContainerProject(
+    scaffoldType: ScaffoldType): Promise<void> {
+  const projectRootPath = getFirstWorkspaceFolderPath();
+  // Check if cmake project
+  const cmakeFile = path.join(projectRootPath, FileNames.cmakeFileName);
+  if (!await FileUtility.fileExists(scaffoldType, cmakeFile)) {
+    const message = `Missing ${
+        FileNames.cmakeFileName} to be configured as Embedded Linux project.`;
+    vscode.window.showWarningMessage(message);
+    throw new CancelOperationError(message);
+  }
+
+  const iotworkbenchprojectFile =
+      path.join(projectRootPath, FileNames.iotworkbenchprojectFileName);
+
+  // Update project host type in IoT Workbench Project file
+  await updateProjectHostTypeConfig(
+      scaffoldType, iotworkbenchprojectFile, ProjectHostType.Container);
+
+  // Update board Id as Raspberry Pi in IoT Workbench Project file
+  const projectConfig =
+      await getProjectConfig(scaffoldType, iotworkbenchprojectFile);
+  projectConfig[`${ConfigKey.boardId}`] =
+      raspberryPiDeviceModule.RaspberryPiDevice.boardId;
+
+  await FileUtility.writeJsonFile(
+      scaffoldType, iotworkbenchprojectFile, projectConfig);
+}
+
+/**
+ * Update project host type configuration in iot workbench project file.
+ * Create one if not exists.
+ * @param type Scaffold type
+ */
+export async function updateProjectHostTypeConfig(
+    type: ScaffoldType, iotWorkbenchProjectFilePath: string,
+    projectHostType: ProjectHostType): Promise<void> {
+  try {
+    if (!iotWorkbenchProjectFilePath) {
+      throw new Error(`Iot workbench project file path is empty.`);
+    }
+
+    // Get original configs from config file
+    const projectConfig =
+        await getProjectConfig(type, iotWorkbenchProjectFilePath);
+
+    // Update project host type
+    projectConfig[`${ConfigKey.projectHostType}`] =
+        ProjectHostType[projectHostType];
+
+    // Add config version for easier backward compatibility in the future.
+    const workbenchVersion = '1.0.0';
+    projectConfig[`${ConfigKey.workbenchVersion}`] = workbenchVersion;
```

Comment:
Do we always override workbench version or update it when it is undefined?

Reply:
The statement is meant to update version if null. Currently only one version. 
Will fix this.

Reply:
Fixed.